.vscode/tasks.json

```
@@ -0,0 +1,36 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "npm: testenv:run",
+      "type": "shell",
+      "command": "npm run testenv:run -- -d",
+      "isBackground": true,
+      "problemMatcher": [
+        {
+          "pattern": [
+            { "regexp": ".", "file": 1, "location": 2, "message": 3 }
+          ],
+          "background": {
+            "activeOnStart": true,
+            "beginsPattern": ".",
+            "endsPattern": "."
+          }
+        }
+      ]
```

I had to make this change to get the background task during debugging working correctly for me:
```suggestion
      "command": "npm run testenv:run",
      "isBackground": true,
      "problemMatcher": {
        "pattern": {
          "regexp": ".",
          "file": 1,
          "location": 2,
          "message": 3
        },
        "background": {
          "activeOnStart": true,
          "beginsPattern": ".",
          "endsPattern": "."
        }
      }
```

Note: the change is to avoid those nested arrays -- those broke in VSCode a year or so ago, and I've had success using this as an alternative. This allows the command to run without the `-- -d` so that the output shows up, and the modified `problemMatcher` runs the command in the background correctly without blocking app startup.